packages/errors/package.json

```diff
@@ -3,8 +3,8 @@
 	"description": "Common error types for Feathers apps",
```

Comment:
Why does this package use tabs and spaces?

Reply:
I was wondering why it looked odd. Fixed in https://github.com/feathersjs/feathers/commit/cbd31c10c2c574de63d6ca5e55dbfb73a5fdd758